README.md

```diff
@@ -0,0 +1,63 @@
+
+# Generic GARDENA smart Gateway MQTT client
+
+This is an unofficial service for the GARDENA smart Gateway allowing the GARDENA
+smart system to be monitored and controlled without the cloud.
+
+## Prerequisites
+
+Enable the community feed as described [here][1].
+
+[1]: https://github.com/easybe/smart-garden-gateway-community
+
+
+## Configuration
+
+A minimal configuration can be written as follows:
+```
+cat << EOF > /etc/sg_mqtt_client.conf
+[mqtt_broker]
+host = my-mqtt-broker.lan
+EOF
+```
+
+Here, everything that can be configured and the default values:
```

Comment:
Not sure what "and the default values" means. Maybe "using the default values"? Or "and their default values"?